app/src/app/[lng]/page.tsx

```diff
@@ -280,7 +289,14 @@ export default function Home({ params: { lng } }: { params: { lng: string } }) {
                           fontWeight="semibold"
                           lineHeight="32"
                         >
-                          700<span className="text-[16px]">Mtco2e</span>
+                          {inventory?.totalEmissions ? (
```

Comment:
Not sure if we should display "In progress" while the inventory progress is not yet 100%? Otherwise the number is sort of incomplete...

Maybe fine for now.